src/mono/mono/metadata/debug-mono-ppdb.c

```diff
@@ -725,6 +724,34 @@ mono_ppdb_lookup_locals (MonoDebugMethodInfo *minfo)
 	return res;
 }
 
+MonoDebugLocalsInfo*
+mono_ppdb_lookup_locals_enc (MonoImage *image, int method_idx)
+{
+	return mono_ppdb_lookup_locals_internal (image, method_idx + 1, TRUE);
```

Comment:
nit: maybe it would be better to store a 1-based offset in `MonoDebugInformationEnc:idx`.